### DISCUSSION TITLE
Install help on Ubuntu focal (20.04)

### DISCUSSION THREAD

**Discussion by cbaci** [hydrogen-music/hydrogen#1028] [Hydrogen on Linux]
Im having difficulty with unmet dependencies after upgrade to 20.04. Anyone have success with this?

**Comment by mauser** [comment]
Hi! 

i suppose it would be the best if you file a bug report at Ubuntu against their Hydrogen package. This sounds like more like a Ubuntu problem then a problem with Hydrogen (we do not provide the Ubuntu package).

**Comment by funkypou** [comment]
Hi cbaci,

No problem here on Kubuntu 20.04 with version 1.0.0-rc1 from the Ubuntu repos.
